run/faasm.py

```diff
@@ -1,104 +0,0 @@
-#!/usr/bin/python3
-import json
-import math
-import re
-import sys
-import time
-
-from os import environ
-from os.path import dirname, join, realpath
-from subprocess import check_output
-
-from faasmcli.util.call import invoke_impl
-
-PROJ_ROOT = dirname(dirname(realpath(__file__)))
```

Comment:
Using a CLI tool like `inv` also avoids having to duplicate constants like `PROJ_ROOT`, `LAMMPS_USER` etc. across files.